app/parse.py

```
@@ -8,8 +26,54 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def get_quotes(url: str) -> list[Quote]:
```

The initial request uses `requests.get(url).content` without checking the response status. Consider capturing the response object and calling `response.raise_for_status()` (or at least checking `response.status_code`) to fail fast on network errors. Also, to be gentler to the site, consider adding a small delay between requests (e.g., `time.sleep(0.2-0.5)`).